src/App.css

```diff
@@ -1,30 +1,16 @@
-body {
-  height: 100vh;
-}
-
-#root {
-  height: 100%
-}
```

Comment:
since we're directly setting the canvas width/height to the window width/height we can lose this